templates/base/README.md.template.mjs

```diff
@@ -25,56 +33,39 @@ Before you begin, you need to install the following tools:
 - Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
 - [Git](https://git-scm.com/downloads)
 
-## Quickstart
-
-To get started with Scaffold-ETH 2, follow the steps below:
-
-1. Clone this repo & install dependencies
-
-```
-git clone https://github.com/scaffold-eth/scaffold-eth-2.git
-cd scaffold-eth-2
-yarn install
-```
-
-2. Run a local network in the first terminal:
-
-```
-yarn chain
-```
+${quickStart[0]}
 
-This command starts a local Ethereum network using Hardhat. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in `hardhat.config.ts`.
-
-3. On a second terminal, deploy the test contract:
-
-```
-yarn deploy
-```
+## Documentation
 
-This command deploys a test smart contract to the local network. The contract is located in `packages/hardhat/contracts` and can be modified to suit your needs. The `yarn deploy` command uses the deploy script located in `packages/hardhat/deploy` to deploy the contract to the network. You can also customize the deploy script.
+Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
 
-4. On a third terminal, start your NextJS app:
+To know more about its features, check out our [website](https://scaffoldeth.io).
 
-```
-yarn start
-```
+## Contributing to Scaffold-ETH 2
 
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
+We welcome contributions to Scaffold-ETH 2!
 
-Run smart contract test with `yarn hardhat:test`
+Please see [CONTRIBUTING.MD](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) for more information and guidelines for contributing to Scaffold-ETH 2.`;
 
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your frontend in `packages/nextjs/pages`
-- Edit your deployment scripts in `packages/hardhat/deploy`
+export default withDefaults(contents, {
+  solidityFrameWork: "",
+  quickStart: `## Quickstart
 
-## Documentation
+To get started with Scaffold-ETH 2, follow the steps below:
 
-Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
+1. Clone this repo & install dependencies
```

Comment:
also, why we're using this text instead of https://github.com/scaffold-eth/scaffold-eth-2/blob/cli/README.md?plain=1#L35

Reply:
Ohhh nice catch! Updated it at https://github.com/scaffold-eth/scaffold-eth-2/pull/782/commits/33a3c654fa74d228c6107a80b2b395ebf56520f7 🙌